pkg/cli/admin/catalog/mirror.go

```diff
@@ -322,10 +324,10 @@ func (o *MirrorCatalogOptions) Complete(cmd *cobra.Command, args []string) error
 		a.Mappings = mappings
 		a.SkipMultipleScopes = true
 		if err := a.Validate(); err != nil {
-			fmt.Fprintf(o.IOStreams.ErrOut, "error configuring image mirroring: %v\n", err)
+			return fmt.Errorf("error configuring image mirroring: %w", err)
```

Comment:
```suggestion
			return fmt.Errorf("error configuring image mirroring: %v", err)
```
Unless you're expecting that a recipient of this error will want to un-wrap it and use either the `.As()` or `e.Err == constantType` constructions to get more info or to have different handling, then it's better to continue to use `%v`.